Makefile

```diff
@@ -51,7 +52,7 @@ harness: tests/harness/harness.pb.go tests/harness/go/go-harness tests/harness/c
 .PHONY: bazel-harness
 bazel-harness:
 	# runs the test harness via bazel
-	bazel run //tests/harness/executor
+	bazel run //tests/harness/executor:executor
```

Comment:
Nit: Not worth blocking merge for 1.5, but the previous form of this is the canonical Bazel label when you have //a/b:b.

Reply:
Ah my bad. Wasn’t sure.